.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
To work locally, we want to ensure that the developer(s) have installed the required engine listed in `package.json`. If not, the `npm` commands won't be executed.